src/components/card/listCard/listCard.css

```diff
@@ -0,0 +1,28 @@
+.list-card {
+    display: flex;
+    justify-content: flex-start;
+    gap: 1rem;
+    width: 70%;
+    margin: 1rem auto;
+    position: relative;
+}
+
+.list-card-img{
+    width: 11rem;
+    height: 6rem;
+    object-fit: cover;
+}
+
+.list-card-details {
+    width: 30rem;
+}
+
+.dropdown-icon {
+    align-self: center;
+    cursor: pointer;
+}
+
+.list-dropdown {
+    /* right:0rem; */
```

Comment:
suggestion : Make sure before your submission of removing commented codes

Reply:
I have removed it. Thanks for pointing out